django/river/common/database_connection/db_connection.py

```diff
@@ -47,7 +48,8 @@ def build_db_url(db_config):
                 "db_config specifies the wrong database model. "
                 "Only 'POSTGRES', 'ORACLE' and 'MSSQL' are currently supported."
             )
-
+        if service_name:
+            return f"{db_handler}://{login}:{password}@{host}:{port}/?service_name={service_name}"
```

Comment:
Can we specify a service name in the mssql case ? If so, I think we'll still need the `url_suffix`

Reply:
it looks like a Orable thing.. I didn't find any info on that with mssql

Reply:
you should add the suffix anyway (even if there are none in the oracle case)